<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Leon</author>
 <class>addBookmarksDialog</class>
 <widget class="QDialog" name="addBookmarksDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>90</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Neues Lesezeichen</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QComboBox" name="comboFolder">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>Ordner</string>
     </property>
     <property name="statusTip">
      <string>Ordner</string>
     </property>
     <property name="whatsThis">
      <string>Ordner des Hinzuzufügenden Lesezeichen</string>
     </property>
     <property name="maxVisibleItems">
      <number>1677777779</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="whatsThis">
      <string>Klicken sie auf Ok wenn sie mit den Angaben zufrieden sind</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineAdress">
     <property name="toolTip">
      <string>Adresse</string>
     </property>
     <property name="statusTip">
      <string>Adresse</string>
     </property>
     <property name="whatsThis">
      <string>Adresse des Hinzuzufügenden Lesezeichen</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Adresse</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineName">
     <property name="toolTip">
      <string>Name</string>
     </property>
     <property name="statusTip">
      <string>Name</string>
     </property>
     <property name="whatsThis">
      <string>Name des Hinzuzufügenden Lesezeichen</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>lineName</tabstop>
  <tabstop>lineAdress</tabstop>
  <tabstop>comboFolder</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addBookmarksDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addBookmarksDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
